### DISCUSSION TITLE
Whitelist domain or address for an alias and block all other domains or addresses

### DISCUSSION THREAD

**Discussion by gogogohub** [simple-login/app#726] [Feature Requests]
I would like to whitelist a domain or address for a particular alias and block all other domains or addresses from sending mail to a particular alias. 

Thanks so much!.

**Comment by IpsmLorem** [comment]
Yes and why not an option to whitelist only the first domain ou adress sending an email too ?
Each alias would also need a better block/allow sender management gui.

**Comment by shinjipons** [comment]
When I create an alias linked to a domain, I would like to only receive emails from that domain.
